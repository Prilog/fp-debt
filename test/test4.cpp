bool satisfy(int x, int y) {
  if (x < 10) {
    if (y > 4) {
      return true;
    } else {
      return false;
    }
  } else {
    return false;
  }
}

int main() {
  int a = 7;
  int b = 7;
  while (satisfy(a, b)) {
    cout << a;
    cout << " ";
    cout << b;
    cout << "\n";
    a = a + 1;
    b = b - 1;
  }
  cout << "finished\n";
  return 0;
}