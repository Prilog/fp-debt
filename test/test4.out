=================
print required action:
"exe" - execute file CDSLResult.hs
*filename* - interpretes CDSLResult.hs into file *filename*
7 7
8 6
9 5
finished
main returned with exit code: 0
